### PR TITLE
Vfx/docs/whats new 12

### PR DESCRIPTION
**Pull request by Vic-Cooper**
Added the new sections included in this draft doc: https://docs.google.com/document/d/1sRf03IdCBZWlTe6m3P6CVOEjcg2TYo88llco2guo9KQ/edit#

### CODE REVIEW

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+This feature helps you create accurate bounds that match their systems so that effects don’t unexpectedly disappear when the camera moves.
+
+For more information, see [Visual effect bounds](visual-effect-bounds.md).
+
+### Graphics/Compute Buffer Support
+
+![img](Images/banner-graphics-compute-buffer-support.png)
+
+VFX version 21.2 also adds support for Graphics/Compute buffers. This makes it easier to handle and transfer large amounts of data to a Visual Effect Graph. This is particularly useful for tracking multiple GameObject positions in your graph.
```
Comment (VladNeykov) [inline com.unity.visualeffectgraph/Documentation~/whats-new-12.md:37]: My bad if I wrote this originally; I think it's misleading saying VFX version 21.2; it should be either VFX version 12.0 or VFX Graph in 2021.2.

Comment (VladNeykov) [reply com.unity.visualeffectgraph/Documentation~/whats-new-12.md:37]: Maybe just Graphics buffers.

From Paul:
> Yes, "Graphics Buffer" it's the most correct way to express it, Structured Buffer is also fine but it refers to the GPU view.

This also showed up in [this ](https://forum.unity.com/threads/compute-buffer-structured-buffer-support-in-unity-2021-2-0b4-3123.1146284/#post-7362275)forum thread.

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+### Signed Distance Field Baker
+
+![img](Images/banner-sdf-baker.png)
+
+This version includes the new Signed Distance Field (SDF) Bake Tool. To access it, navigate to **Window > Visual Effects > Utilities > SDF Bake Tool**. You can use this tool to quickly turn meshes and prefabs into SDF assets which you can use to create various effects, such as custom collisions, or to make particles conform to a particular shape.
+
+For more information, see [Signed Distance Fields in the Visual Effect Graph](sdf-in-vfx-graph.md).
+
+### HDRP Decal Output
```
Comment (VladNeykov) [inline com.unity.visualeffectgraph/Documentation~/whats-new-12.md:49]: The decal output section looks to be empty (image and description)

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Updated
+
+The following is a list of improvements Unity made to the Visual Effect Graph in version 12.0, embedded in Unity 2021.2. Each entry includes a summary of the improvement and, if relevant, a link to any documentation.
+
+### Transform Shape
+
+![img](Images/banner-transform-shape.gif)
+
+This VFX graph version adds a transform to various shape blocks and operators to make them easier to use and to give you more precise control. For example, you can use this transform to rotate a Position Circle block to make the circle shape face a new direction, or to scale the shape of a sphere collision block on one axis to collide with an ellipse.
```
Comment (VladNeykov) [inline com.unity.visualeffectgraph/Documentation~/whats-new-12.md:60]: Just googled it and apparently ellipsoid is the 3D version of an ellipse. Apologies, I wrote the wrong one originally, should be "ellipsoid".

**Review by VladNeykov — CHANGES_REQUESTED** [review]
Hi Vic, looking good!
I've noted a few possible things to check out.

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+This version of the Visual Effect Graph adds support for lit outputs in the Universal Render Pipeline (URP). You can use this to create effects that can respond to the lighting in the scene.
+
+### 2D Renderer Support (Compute Capable Devices)
+
+![img](Images/banner-2d-renderer-support.png)
+
+In this version, the Visual Effect Graph has added support for the Universal Render Pipeline’s (URP) 2D Renderer. This means that you can now render effects in a 2D project and sort them along with sprites in your scene.
+
+For more information, see [Rendering in the Universal Render Pipeline](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest?/manual/rendering-in-universalrp.html).
```
[the remark this answers is not in the document]
Comment (Vic-Cooper) [reply com.unity.visualeffectgraph/Documentation~/whats-new-12.md:21]: Ah, you make a good point, the equivalent 12.x link would be better! Done :)

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md (hunk 2 above)
Comment (Vic-Cooper) [reply com.unity.visualeffectgraph/Documentation~/whats-new-12.md:49]: ops. Added!

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md (hunk 3 above)
Comment (Vic-Cooper) [reply com.unity.visualeffectgraph/Documentation~/whats-new-12.md:60]: Ah, that makes more sense! Fixed.

File: com.unity.visualeffectgraph/Documentation~/whats-new-12.md (hunk 1 above)
Comment (Vic-Cooper) [reply com.unity.visualeffectgraph/Documentation~/whats-new-12.md:37]: Thanks! Fixed.

**Review by Vic-Cooper — COMMENTED** [review]
Reviewed feedback, pushing changes accordingly.

**Comment by Vic-Cooper** [comment]
@VladNeykov Pushed changes to fix the issues you mentioned in your comments.

**Review by VladNeykov — APPROVED** [review]
Looking great, thanks for the PR @Vic-Cooper ! :)

**Pull request merged**